src/mono/wasm/debugger/tests/debugger-test/debugger-test.csproj

```diff
@@ -19,7 +19,7 @@
 
   <Target Name="PrepareForWasmBuildApp" DependsOnTargets="RebuildWasmAppBuilder;Build">
     <PropertyGroup>
-      <WasmMainAssemblyPath>$(OutDir)debugger-test.dll</WasmMainAssemblyPath>
+      <WasmMainAssemblyFileName>debugger-test.dll</WasmMainAssemblyFileName>
```

Comment:
Does `TargetFileName` not default to `debugger-test.dll` here?

Reply:
It will. Good catch 👍